src/VisualStudio/IntegrationTest/IntegrationTests/IntegrationTestSourceGenerator.cs

```diff
@@ -13,7 +13,7 @@ internal sealed class IntegrationTestSourceGenerator : ISourceGenerator
     {
         public const string GeneratedClassName = nameof(IntegrationTestSourceGenerator) + "Output";
 
-        public void Initialize(InitializationContext context)
+        public void Initialize(GeneratorInitializationContext context)
```

Comment:
Regardless of the obsoletion error, how the compiler accepted `InitializationContext` as a parameter while the interface defines `GeneratorInitializationContext`?